src/boss/cloudvm/local_worker.go

```diff
@@ -0,0 +1,99 @@
+package cloudvm
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/open-lambda/open-lambda/ol/common"
+)
+
+// WORKER IMPLEMENTATION: LocalWorker
+type LocalWorkerPoolPlatform struct {
+	// no platform specific attributes
+}
+
+func NewLocalWorkerPool() *WorkerPool {
+	return &WorkerPool{
+		WorkerPoolPlatform: &LocalWorkerPoolPlatform{},
+	}
+}
+
+func (_ *LocalWorkerPoolPlatform) NewWorker(workerId string) *Worker {
+	return &Worker{
+		workerId: workerId,
+		workerIp: "",
+	}
+}
+
+func (_ *LocalWorkerPoolPlatform) CreateInstance(worker *Worker) {
+	log.Printf("Creating new local worker: %s\n", worker.workerId)
+
+	// Initialize the worker directory if it doesn't exist
+	initCmd := exec.Command("./ol", "worker", "init", "-p", worker.workerId, "-i", "ol-min")
```

Comment:
The reasons are a little strange, but it will be much faster if we start the worker detached: https://github.com/open-lambda/open-lambda/blob/main/src/worker/commands.go#L93